### PR TITLE
rzshell_which.py: Ignore FAKE and INNER entries

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [X] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This pr makes `rzshell_which.py` ignore `RZ_CMD_DESC_TYPE_FAKE` and `RZ_CMD_DESC_TYPE_INNER` entries since they don't have command handlers (even if one is set).

**Test plan**

All builds are green.

**Closing issues**

...

### CODE REVIEW

**Comment by wargio** [comment]
i don't think we should add this to the testing suite. it's just for searching a command from the dev pov.

**Comment by kazarmy** [comment]
ok a2cfc9700d2e94cdd6b473469887bac71155d504 ... the test was a bit slow anyway

File: librz/core/cmd_descs/rzshell_which.py [hunk 1]
```diff
@@ -29,7 +35,11 @@ def find_entry(commands, rzcommand):
             return None
 
         if c["name"] == rzcommand:
-            return c
+            if "type" in c:
+                if c["type"] not in [CD_TYPE_FAKE, CD_TYPE_INNER]:
+                    return c
+            else:
+                return c
```
Comment (ret2libc) [inline librz/core/cmd_descs/rzshell_which.py]:
```suggestion
            if c.get("type") not in [CD_TYPE_FAKE, CD_TYPE_INNER]:
                return c
```

Comment (kazarmy) [reply librz/core/cmd_descs/rzshell_which.py]: Thanks!

**Review by ret2libc — APPROVED** [review]

**Pull request merged**